new-frontiers-14/frontier-station-14
# PTK-800 Require Power

## About the PR
PTK-800 Now require 1500 power to work.

This PR also has a cleanup code for the thrusters code I made.

## Why / Balance
Why were they using no power?

## How to test
Spawn one in, you can now toggle its power on and off and it require cable connection.

Note when working its using the required power (1500 from yml), when its off it using 1, it cannot use 0 since the apc code will go crazy thinking it's no longer required

## Media
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame,

![image](https://github.com/new-frontiers-14/frontier-station-14/assets/39403717/484cf848-4947-4360-b0d6-31affa79e1a6)

![image](https://github.com/new-frontiers-14/frontier-station-14/assets/39403717/4e97ac58-b534-4f9e-af89-533715c44041)

![image](https://github.com/new-frontiers-14/frontier-station-14/assets/39403717/713a944c-1029-4309-b4c8-d10689cf5c16)

## Breaking changes
N/A

**Changelog**
- tweak: PTK-800 Now require 1500 power to shot, but can be turned off at any time to save power.
